shell/common/shell.cc

```diff
@@ -186,30 +186,33 @@ std::unique_ptr<Shell> Shell::CreateShellOnPlatformThread(
                     !settings.skia_deterministic_rendering_on_cpu),
                 is_gpu_disabled));
 
+  // Create the platform view on the platform thread (this thread).
+  auto platform_view = on_create_platform_view(*shell.get());
```

Comment:
Just to double-check, will `Shell::OnPlatformViewCreated` still work as expected and call `Rasterizer::Setup` with the created platform view?

Reply:
Yes

Reply:
Basically we're dealing with an artifact of targeting OpenGL first - in GL you need a surface to do rendering, but in Metal and Vulkan you don't. Even in GL we can just create pbuffer surfaces to do the rendering when we need to.